public/js/show-more-comment.js

```diff
@@ -0,0 +1,36 @@
+window.addEventListener("load", init);
+
+function init(){
+    let btn = document.querySelector("#show-more");
+    let loader = document.querySelector(".loader");
+    let span = document.querySelector(".content-span");
+
+    btn.addEventListener("click", function (e) {
+        btn.style.visibility = "hidden";
+        loader.style.visibility = "visible";
+
+        let comments = document.querySelectorAll(".js-card");
+        let lastCom = comments[comments.length - 1].getAttribute("id");
+        const Params = new URLSearchParams();
+        Params.append("comId", lastCom );
+        const Url = new URL(window.location.href);
+
+        fetch(Url.pathname + "?" + Params.toString() + "&loadCom=1", {
+            headers: {
+                "X-Requested-With": "XMLHttpRequest"
+            }
+        }).then(response => response.json()
+        ).then(data => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/VinzOo93/P6SnowTricks/pullRequest?prid=8761575)
```suggestion
        ).then((data) => {
```